app/build.gradle

```diff
@@ -65,6 +65,7 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            unitTests.returnDefaultValues = true
```

Comment:
why is this needed ?

Reply:
we don't, I remember removing it. It's a dangerous to use:
Caution: Setting the returnDefaultValues property to true should be done with care. The null/zero return values can introduce regressions in your tests, which are hard to debug and might allow failing tests to pass.